HW0/P4/P4.py

```diff
@@ -52,11 +79,48 @@
     # a = ?
     # s = ?
 
+    #make a 6 x K matrix to store
+    S = np.zeros([6,K])
+    S = np.matrix(S)
+
+    # make A
+    A = np.zeros([6,6])
+    A = np.matrix(A)
+    A[0,0] = 1
```

Comment:
It would have been more efficient and more elegant to use a for loop to fill in the values of the array A: 
 for i in range(3):
         A[i][i]=1   
 for i in range(3,6):
        A[i][i]=1-dt*c
        A[i-3][i]=dt
        
A=np.asmatrix(A)